File: C3/C3 08 AS 03 Loops/02.cpp
//02. Write a C++ program to print first N natural numbers in reverse order.
#include<iostream>
using namespace std;
int main()
{
    int n;
    cout<< "Enter a number: ";
    cin>>n;
    for(n;n;n--)
        cout<< n << " ";
    return 0;
}

File: Python/PY 25 AS 11 more on while/02.py
#   02. Write a python script to calculate sum of squares of first N natural numbers.
n = int(input("Enter a number: "));
print("Sum of square of first %d natural number is: "%(n),end="");
sum = 0;
while n:
    sum=sum+(n*n);
    n-=1;
print(sum);